packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8591,12 +8597,65 @@ export class Editor extends EventEmitter<TLEventMap> {
 	}
 
 	/** @deprecated Use {@link Editor.getSvgString} or {@link Editor.getSvgElement} instead. */
-	async getSvg(shapes: TLShapeId[] | TLShape[], opts: TLImageExportOptions = {}) {
+	async getSvg(shapes: TLShapeId[] | TLShape[], opts: TLSvgExportOptions = {}) {
 		const result = await this.getSvgElement(shapes, opts)
 		if (!result) return undefined
 		return result.svg
 	}
 
+	/**
+	 * Get an exported image of the given shapes.
+	 *
+	 * @param shapes - The shapes (or shape ids) to export.
+	 * @param opts - Options for the export.
+	 *
+	 * @returns A blob of the image.
+	 * @public
+	 */
+	async toImage(shapes: TLShapeId[] | TLShape[], opts: TLImageExportOptions = {}) {
+		const withDefaults: TLImageExportOptions = {
+			format: 'png',
+			scale: 1,
+			pixelRatio: opts.format === 'svg' ? undefined : 2,
```

Comment:
feels like this `opts.format === 'svg' ? undefined : 2,` bit of logic should be in deeper in `getSvgString`/`getSvgAsImage` to avoid footguns/forgetfulness

Reply:
I can't push this down further because those methods don't know what format they're exporting to - they're just generating an svg, so they generate SVG style. I guess i could push these down further, but `getSvg({format: 'png'})` feels a bit weird/confusing to me

Reply:
oh, i see - i keep forgetting that `format` is optional.